src/external/rawspeed/data/cameras.xml

```diff
@@ -3231,24 +3231,7 @@
 			<Color x="1" y="1">BLUE</Color>
 		</CFA>
 		<Crop x="0" y="0" width="0" height="0"/>
-		<Sensor black="200" white="4095"/>
-		<Hints>
-			<Hint name="nikon_override_auto_black" value=""/>
```

Comment:
Hmm.
I guess i "broke" it in 6810787e316f3a29af2b32d63529c34efec57e36, essentially deprecated it.

Now, will need to check which cameras have  black != "0" and no `nikon_override_auto_black` (as of before that commit ^) :/
If there are any, i suspect they will now have wrong black level.

And then drop all `<Hint name="nikon_override_auto_black" value=""/>`